### DISCUSSION TITLE
"Заказать демо" на основном сайте отвалилось

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#421] [ОШБК!] [answered]
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/4cf4363d-c714-4e8d-80f5-b4f54439b2b0)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/71624171/8359200f-b3e1-44b1-ac23-5de010a08023)

https://ru.totum.online/downloaddemo

очень прошу починить по возможности)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Старый демо потчил в бозе. Вместо него cloud, но там можно заказать с примерами из учебного курса. На ссылку поставил редирект. До документации еще доберемся — просто все это собиралось 3 года, сложно за неделю все переделать.
